public_desktop/desktop-app.js

```diff
@@ -154,6 +154,47 @@ try {
 	ipc.on( 'is-calypso', function() {
 		ipc.send( 'is-calypso-response', document.getElementById( 'wpcom' ) !== null );
 	} );
+
+	ipc.on( 'app-config', function( event, config, debug, details ) {
+
+		// if this is the first run, show Windows and Mac users a pin app reminder
+		if ( details.firstRun ) {
+		
+			var container = document.querySelector( '#wpcom' );
+			var pinApp = container.querySelector( '.pin-app' );
+	
+			if ( ! pinApp ) {
+				var node = document.createElement( 'div' );
+				node.className = 'pin-app';
+				container.appendChild( node );
+				pinApp = container.querySelector( '.pin-app' );
+			}
+
+			var closeButton = '<a href="#" class="pin-app-close"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17.705,7.705l-1.41-1.41L12,10.59L7.705,6.295l-1.41,1.41L10.59,12l-4.295,4.295l1.41,1.41L12,13.41 l4.295,4.295l1.41-1.41L13.41,12L17.705,7.705z"/></svg></a>';
+			var pinAppMsg = "";
+
+			if ( details.platform == "windows" ) {
+				pinAppMsg = "<h2>Keep WordPress.com in your taskbar</h2>" + 
+				"<p>Drag the icon from your desktop to your taskbar</p>" +
+				'<img src="/desktop/pin-app-taskbar.png" alt="" width="143" height="27" />';
+			} else if ( details.platform == "darwin" && !details.pinned ) {
+				pinAppMsg = "<h2>Keep WordPress.com in your dock</h2>" +
+				"<p>Right-click the WordPress.com icon, select Options > Keep in Dock</p>" +
```

Comment:
"Right-click the WordPress.com icon in your dock" (otherwise it's not clear which icon you should click.